vrc-get-gui/locales/zh_cn.json5

```diff
@@ -94,6 +94,8 @@
     "projects:manage:button:select repositories": "选择存储库",
     "vpm repositories:source:official": "官方",
     "vpm repositories:source:curated": "精选",
+    "projects:manage:suggest resolve": "项目所需的某些软件包未安装。强烈建议修复软件包缺失问题。",
+    "projects:manage:button:resolve": "修复",
```

Comment:
could you update key based on this discussion? https://github.com/vrc-get/vrc-get/discussions/894#discussioncomment-9394364